crates/aptos-telemetry/src/sender.rs

```diff
@@ -92,8 +93,17 @@ impl TelemetrySender {
 
         let token = self.get_auth_token().await?;
 
-        let scraped_metrics =
-            prometheus::TextEncoder::new().encode_to_string(&registry.gather())?;
+        let mut gathered_metrics = registry.gather();
+
+        // This will add a synthetic `up 1` metric to the pushed metrics.
+        // In prometheus' pull model this metric is normally added by the scraping daemon (https://prometheus.io/docs/concepts/jobs_instances/#automatically-generated-labels-and-time-series).
+        // Adding it here as synthetic metric allows us to use this metric in dashboards to show the set of nodes without filtering on a specific metric.
+        // Note that this is just an approximation.
+        // In prometheus push model this value can be 0 when a discovered target couldn't be scraped (e.g. shortly after a crash etc.),
+        // whereas the synthetic up metric is either `1` or completely absent but never 0.
```

Comment:
I think this metric should be called something else other than `up`. Is this named to be maintain compatibility for dashboards?